app/assets/stylesheets/layout/navbar.css.scss

```diff
@@ -0,0 +1,87 @@
+nav { 
+    color: white;
```

Comment:
white is a standard web color meaning rgb(255,255,255), I don't think turning it into a variable adds clarity, rather, it detracts clarity as I could assign whatever value I want to $white